test/presubmit-tests-with-pipeline-deployment.sh

```diff
@@ -73,6 +77,14 @@ source "${DIR}/deploy-kubeflow.sh"
 # Install Argo CLI and test-runner service account
 source "${DIR}/install-argo.sh"
 
+# When project is not ml-pipeline-test, VMs need permission to fetch some images in gcr.io/ml-pipeline-test.
```

Comment:
what are the images? is it just gcr.io/ml-pipeline-test/image-builder?

Reply:
Yes, I think so. image-builder is the error message I got.

Reply:
You should be able to copy(retag) the image to your own project's GCR registry. Adding permission is not a scalable approach as there are quota limits on IAM policy per project.

Reply:
Thanks for the advice. Let me try copying instead.

Reply:
Done